downstreamadapter/dispatcher/dispatcher.go

```diff
@@ -254,7 +254,6 @@ func (d *Dispatcher) HandleEvents(dispatcherEvents []DispatcherEvent, wakeCallba
 	for _, dispatcherEvent := range dispatcherEvents {
 		log.Debug("dispatcher receive all event",
 			zap.Stringer("dispatcher", d.id),
-			zap.String("eventType", commonEvent.TypeToString(dispatcherEvent.Event.GetType())),
 			zap.Any("event", dispatcherEvent.Event))
```

Comment:
event type can be know from the event field, so no need to print this, duplicate field

Reply:
Please restore this change. The eventType printed in zap.Any("event", dispatcherEvent.Event) is of int type and is not human - readable.

Reply:
I found that it's human-readable, since the `event` first part is something like `ResolvedEvent`.